src/ai/backend/manager/registry.py

```diff
@@ -2033,7 +2038,9 @@ async def _update() -> None:
                         )
 
                     if kernel["agent_addr"] is None:
-                        await self.mark_kernel_terminated(kernel["id"], "missing-agent-allocation")
+                        await self.mark_kernel_terminated(
```

Comment:
Same here! Please update type definition of `mark_kernel_terminated()` function's `reason` argument.